src/pages/Vault/index.tsx

```diff
@@ -147,16 +167,51 @@ const Vault = (): JSX.Element => {
           setCapacity(issuableAmount.value);
         }
       } catch (error) {
-        console.log('[VaultDashboard React.useEffect] error.message => ', error.message);
+        console.log('[Vault React.useEffect] error.message => ', error.message);
       }
     })();
   }, [
     bridgeLoaded,
     dispatch,
-    selectedVaultAddress
+    vaultAccountId
   ]);
 
-  const VAULT_ITEMS = [
+  const {
+    isIdle: governanceRewardIdle,
+    isLoading: governanceRewardLoading,
+    data: governanceReward,
+    error: governanceRewardError
+  } = useQuery<MonetaryAmount<Currency<GovernanceUnit>, GovernanceUnit>, Error>(
+    [
+      GENERIC_FETCHER,
+      'vaults',
+      'getGovernanceReward',
+      vaultAccountId,
+      COLLATERAL_ID_LITERAL,
+      GOVERNANCE_TOKEN_SYMBOL
```

Comment:
@theodorebugnet 
Could you please review if I used the correct arguments?

Reply:
Looks right to me